examples/users/hydra.yaml

```diff
@@ -0,0 +1,11 @@
+connection:
```

Comment:
I like this. Original hydra (basically) just had a root level `uri`, but this is more extedable